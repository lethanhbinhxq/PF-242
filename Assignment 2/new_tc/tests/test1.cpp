#include "hcmcampaign.h"
using namespace std;

int main()
{

    cout << "----- Sample Testcase 01 -----" << endl;
    Configuration *config = new Configuration("tests/initial_config.txt");
    cout << config->str() << endl;
    delete config;
    return 0;
}
