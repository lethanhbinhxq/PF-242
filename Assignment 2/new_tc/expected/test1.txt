----- Sample Testcase 01 -----
[num_rows=10,num_cols=8,arrayForest=[(1,2),(3,5)],arrayRiver=[(0,0),(0,4)],arrayFortification=[(6,6)],arrayUrban=[(2,0)],arraySpecialZone=[(9,7)],liberationUnits=[Vehicle[vehicleType=TANK,quantity=5,weight=2,position=(1,2)],Infantry[infantryType=REGULARINFANTRY,quantity=5,weight=2,position=(1,1)]],ARVNUnits=[Vehicle[vehicleType=TANK,quantity=5,weight=2,position=(3,2)],Infantry[infantryType=REGULARINFANTRY,quantity=5,weight=2,position=(3,3)]],eventCode=23]